bd_spot_wrapper/spot_wrapper/spot.py

```diff
@@ -306,8 +357,20 @@ def get_image_responses(self, sources, quality=None):
                 quality = [quality] * len(sources)
             else:
                 assert len(quality) == len(sources)
-            sources = [build_image_request(src, q) for src, q in zip(sources, quality)]
-            image_responses = self.image_client.get_image(sources)
+            img_requests = [
```

Comment:
What do you mean by `pixel_format`?

Reply:
It represents if the image should be rgb or grayscale